great_expectations/util.py

```diff
@@ -81,7 +81,9 @@
     import sqlalchemy as sa
     from sqlalchemy import Table
     from sqlalchemy.engine import reflection
+    from sqlalchemy.exc import RemovedIn20Warning
```

Comment:
Can you get rid of this import now that you've moved to `great_expectations/df_to_database_loader.py`?

Reply:
Thank you for catching this 🙇🏼 too